internal/bminventory/inventory.go

```diff
@@ -2289,6 +2289,28 @@ func (b *bareMetalInventory) DownloadHostLogs(ctx context.Context, params instal
 	return filemiddleware.NewResponder(installer.NewDownloadHostLogsOK().WithPayload(respBody), downloadFileName, contentLength)
 }
 
+func (b *bareMetalInventory) DownloadClusterLogs(ctx context.Context, params installer.DownloadClusterLogsParams) middleware.Responder {
+	log := logutil.FromContext(ctx, b.log)
+	log.Infof("Downloading logs from cluster %s", params.ClusterID)
+	_, err := b.getCluster(ctx, params.ClusterID.String())
+	if err != nil {
+		return common.GenerateErrorResponder(err)
+	}
+
+	files, err := b.objectHandler.ListObjectsByPrefix(ctx, fmt.Sprintf("%s/logs/", params.ClusterID))
```

Comment:
Why not have an objectHandler function of `DownloadFilesByPrefix()`?

Reply:
hmm thought about it, dunno . We can. But still nice to have list by prefix one too

Reply:
+1 to @ronniel1